common/src/main/java/com/wynntils/utils/render/RenderUtils.java

```diff
@@ -956,7 +956,7 @@ public static void rotatePose(PoseStack poseStack, float centerX, float centerZ,
     }
 
     // Basically this is ItemRenderer#renderGuiItem, but we can modify the poseStack
-    private static void renderGuiItem(ItemStack itemStack, int x, int y, float scale) {
+    private static void renderGuiItem(PoseStack poseStack, ItemStack itemStack, int x, int y, float scale) {
```

Comment:
I don't get this. What does this do?

Reply:
We can now pass a pose stack. In many screens, we use pose stack translations to set a drawing origin. The old impl did not allow us to use an existing pose to draw an item, so we had to figure out x, y, which is quite messy. Now, we can support pose translations (but not scaling)

Reply:
Also, this is completely rewritten in the 1.19.4 port. Something to consider.

Reply:
Line 974. If it is rewritten, I can adapt the code accordingly.

Reply:
Ah, I missed that.